blog/_posts/2016-9-19-our-2nd-blog-post.md

```diff
@@ -0,0 +1,24 @@
+---
+title: The Miracle
+layout: post
+created_at: 2016-09-19
+permalink: blog/2016-09-19-The-Miracle
+current: blog
+author: Nada Ashraf and Mayar Alaa
+twitter: RubysSecretteam
```

Comment:
You are missing the post category (it should be `student-posts`). Please add it to the metadata, following the same format as e.g. [here](https://raw.githubusercontent.com/rails-girls-summer-of-code/summer-of-code/gh-pages/blog/_posts/2016-08-24-hello-from-team-Crackers.md)